src/libpriv/rpmostree-postprocess.c

```diff
@@ -959,9 +959,6 @@ rpmostree_postprocess_final (int            rootfs_dfd,
 {
   GLNX_AUTO_PREFIX_ERROR ("Finalizing rootfs", error);
 
-  if (!ror_compose_postprocess_final (rootfs_dfd, error))
```

Comment:
Ah forgot to mention in the commit message, this was placed incorrectly in the previous patch too - see the comment right below.